### ISSUE TITLE
Improve Lighten/Darken tools

### ISSUE THREAD

**Issue by c-sanchez** [bgrabitmap/lazpaint#132]
See how the Darken/Lighten tool is in LazPaint and Phoxo
lazpaint 
![lazpaint lighten/darken demo1](https://i.ibb.co/s3DWkhd/lazpaint-lighten-darken-tool.gif)
![phoxo lighten/darken demo1](https://i.ibb.co/rMy0zpD/phoxo-lighten-darken-tool.gif)

lazpaint
![lazpaint lighten/darken demo2](https://i.ibb.co/jJHwWDD/lp-demo1.gif)
phoxo
![phoxo lighten/darken demo2](https://i.ibb.co/vXCYGMr/px-demo1.gif)

Forum topic: https://forum.lazarus.freepascal.org/index.php/topic,12037.msg354837.html#msg354837

**Comment by circular17** [comment]
Yeah I thought about it as well. That would need some new development and translation so I probably will keep it for next release.

Also I am thinking of adding an erash brush, similar to the brush tool but that erases/lightens/darkens.

**Comment by c-sanchez** [comment]
an erashing brush sounds interesting
